src/d3a/setup/kpi/d3asim_3464_young_couple.py

```diff
@@ -44,7 +44,7 @@ def get_setup(config):
                                 fit_to_limit=True)),
                          ]),
                     Area("Community PV", strategy=PVStrategy(
-                        max_panel_power_W=4000, panel_count=1,
+                        capacity_kW=4000, panel_count=1,
```

Comment:
Shouldn't this be divided by 1000?

Reply:
It probably doesn't change much, unless some researcher has based their computations on this.

Reply:
Yeah, I skipped adapting all power values in all setup files and only adapted the ones that are used in the integrationtests which failed. There are a lot of setup files that are not used any more. 
Do you think I should take the time to adapt the power values in all of them?

Reply:
I think it's not necessary, as long as the values are internally coherent with each other. The only small issue I see is that 4000kW is an extremely high value for a solar panel (about 1000 times bigger than what you would expect :P )

Reply:
It was actually the only place where the power value was so high. SO I converted it to kW.